snforge_std/src/cheatcodes.cairo

```diff
@@ -144,6 +162,9 @@ fn start_mock_call<T, impl TSerde: core::serde::Serde<T>, impl TDestruct: Destru
     handle_cheatcode(cheatcode::<'mock_call'>(inputs.span()));
 }
 
+/// Cancels the `mock_call` / `start_mock_call` for the function `function_selector` of a contract at the given address.
+/// - `contract_address` - target contract address
+/// - `function_selector` - selector of the function
```

Comment:
![image](https://github.com/foundry-rs/starknet-foundry/assets/65187002/f9013598-dcba-4e9a-9c51-fd0f8778bce2)

Reply:
love that meme